raven/contrib/flask.py

```diff
@@ -270,7 +277,11 @@ def init_app(self, app, dsn=None, logging=None, level=None,
             if self.logging_exclusions is not None:
                 kwargs['exclude'] = self.logging_exclusions
 
-            setup_logging(SentryHandler(self.client, level=self.level), **kwargs)
+            handler = SentryHandler(self.client, level=self.level)
+            setup_logging(handler, **kwargs)
+            if logging_filters is not None:
+                for _filter in logging_filters:
```

Comment:
I remember now...`filter` is a function... so I just avoided that. Are you ok with `filter` as a name here or should I use `f` or something?